Update conclusion episode

SarahAlidoost opened the pull request:
closes #16

SarahAlidoost committed:
add qustions and objective

SarahAlidoost committed:
update text, add exercise and figures

SarahAlidoost committed:
fix a minor thing

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
 objectives:
-- "First learning objective. (FIXME)"
+- "Relate each FAIRification practice with FAIR principles."
+- "Know about available tools/resources for assessing FAIRness of your data."
```
```suggestion
- "Be aware of available tools/resources for assessing FAIRness of your data."
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
-- "First key point. Brief Answer to questions. (FIXME)"
+- "FAIRification practices are about documentation, metadata, file formats, access to data,
+persistent identifiers, and data licenses."
+- "FAIR will change the way we do science."
```
I find this keypoint a bit too dreamy, how about:

```suggestion
- "The FAIR principles provide clear handles on data management in the movement towards open and sustainable science."
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+- "FAIRification practices are about documentation, metadata, file formats, access to data,
+persistent identifiers, and data licenses."
+- "FAIR will change the way we do science."
+- "FAIR brings all the stakeholders together."
```
I guess here you want to say something about how the FAIR principles affect everyone, from top to bottom, how about:

```suggestion
- "The FAIR principles promote maximum use of research data, involving all the stakeholders from data producers to funding agencies."
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+
+> ## You make your research data findable by:
+>
+> ![findable]({{ page.root }}/fig/findable.jpg)
```
Do we not have higher resolution images? These are a bit grainy.

stefsmeets replied:
I like the figures, but I'm not a huge fan of this layout. It would be nice if we could have the figure on the left, and then the points right next to them, but I guess that is difficult to do with the template. Howe about we just have a row of these images at the top or the bottom of this section?

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> ![findable]({{ page.root }}/fig/findable.jpg)
+>
+> - publishing your **data** and/or **metadata** in a searchable resource such as a repository
```
```suggestion
> - Publish your **data** and/or **metadata** in a searchable resource such as a repository
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+persistent identifiers, and data licenses.
+We also explained what a FAIRification practice means in a FAIR context. In summary:
+
+> ## You make your research data findable by:
```
```suggestion
> ## Make your research findable
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> - publishing your **data** and/or **metadata** in a searchable resource such as a repository
+>   that assigns a **persistent identifier**.
+> - including rich accurate machine-readable descriptive **metadata** and keywords to your data,
```
```suggestion
> - Include rich accurate machine-readable descriptive **metadata** and keywords to your data,
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+{: .checklist}
+
+> ## You make your research data accessible by:
```
```suggestion
> ## Make your data acessible
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> ![accessible]({{ page.root }}/fig/accessible.jpg)
+>
+> - attaching a **data license** or clear data accessibility statement in your openly available administrative metadata.
```
```suggestion
> - Attach a **data license** or clear data accessibility statement in your openly available administrative metadata.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+> ![accessible]({{ page.root }}/fig/accessible.jpg)
+>
+> - attaching a **data license** or clear data accessibility statement in your openly available administrative metadata.
+> - ensuring your data are archived in long-term storage and retrievable by their **persistent identifier** using a standard protocol.
```
```suggestion
> - Ensure your data are archived in long-term storage and retrievable by their **persistent identifier** using a standard protocol.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> - attaching a **data license** or clear data accessibility statement in your openly available administrative metadata.
+> - ensuring your data are archived in long-term storage and retrievable by their **persistent identifier** using a standard protocol.
+> - giving **access** to the **metadata**, even if the data are closed.
```
```suggestion
> - Give **access** to the **metadata**, even if the data are closed.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+> - giving **access** to the **metadata**, even if the data are closed.
+{: .checklist}
+
+> ## You make your research data interoperable by:
```
```suggestion
> ## Make your data interoperable
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> ![interoperable]({{ page.root }}/fig/interoperable.jpg)
+>
+> - including sufficient and structural **metadata** in accordance with your research community’s standards.
```
```suggestion
> - Include sufficient and structural **metadata** in accordance with your research community’s standards.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+> ![interoperable]({{ page.root }}/fig/interoperable.jpg)
+>
+> - including sufficient and structural **metadata** in accordance with your research community’s standards.
+> - using common standards, terminologies, vocabularies, and ontologies for the data.
```
```suggestion
> - Use common standards, terminologies, vocabularies, and ontologies for the data.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> - including sufficient and structural **metadata** in accordance with your research community’s standards.
+> - using common standards, terminologies, vocabularies, and ontologies for the data.
+> - preferring open, long-term viable **file formats** for your data and metadata.
```
long-term viable -> sustainable?

```suggestion
> - Prefer open, long-term viable **file formats** for your data and metadata.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+> - preferring open, long-term viable **file formats** for your data and metadata.
+{: .checklist}
+
+> ## You make your research data reusable by:
```
```suggestion
> ## Make your data reusable
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> ![reusable]({{ page.root }}/fig/reusable.jpg)
+>
+> - attaching all the relevant contextual information in either the **documentation** or **metadata**.
```
```suggestion
> - Attach all the relevant contextual information in either the **documentation** or **metadata**.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+> ![reusable]({{ page.root }}/fig/reusable.jpg)
+>
+> - attaching all the relevant contextual information in either the **documentation** or **metadata**.
+> - including sufficient and structural **metadata** in accordance with your research community’s standards.
```
```suggestion
> - Include sufficient and structural **metadata** in accordance with your research community’s standards.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+>
+> - attaching all the relevant contextual information in either the **documentation** or **metadata**.
+> - including sufficient and structural **metadata** in accordance with your research community’s standards.
+> - preferring open, long-term viable **file formats** for your data and metadata.
```
```suggestion
> - Prefer open, long-term viable **file formats** for your data and metadata.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+> - attaching all the relevant contextual information in either the **documentation** or **metadata**.
+> - including sufficient and structural **metadata** in accordance with your research community’s standards.
+> - preferring open, long-term viable **file formats** for your data and metadata.
+> - applying a machine-readable **data license**.
```
```suggestion
> - Apply a machine-readable **data license**.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+
+## Assessing FAIRness
+
+FAIR is a journey. A FAIR assessment now is a snapshot in time.
```
```suggestion
FAIR is a journey. The FAIR principles are guidelines. A FAIR assessment now is a snapshot in time.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+- [FAIR-Aware](https://fairaware.dans.knaw.nl/) assessment tool.
+- [Zenodo overview](https://about.zenodo.org/principles/) of how the service responds to the FAIR principles.
+- A list of resources for the assessment of digital objects provided by [FAIRassist](https://fairassist.org/#!/).
+- [Mirror, mirror on the wall, who is the FAIRest one of all?](https://forschungsdaten-thueringen.de/entry/mirror-mirror-on-the-wall-tkfdm-announces-fairest-dataset-award.html).
```
'Mirror, mirror on the wall, who is the FAIRest one of all?'
🤦‍♂️

```suggestion
- [Mirror, mirror on the wall, who is the FAIRest one of all?](https://forschungsdaten-thueringen.de/entry/mirror-mirror-on-the-wall-tkfdm-announces-fairest-dataset-award.html)
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+We provided recommendations, examples of steps you can take before, during and after your research project
+to make your research data more FAIR. FAIR sounds like a lot of work. However, you can get started by:
+
+- understanding how FAIR translates to your field/community.
+- using standards that are common to your field/community.
+- collaborating with other communities to define a relevant metadata scheme/standards.
+- checking the requirements of the repository where data is stored.
```
```suggestion
- Understand how FAIR translates to your field/community.
- Use standards that are common to your field/community.
- Collaborate with other communities to define a relevant metadata scheme/standards.
- Check the requirements of the repository where data is stored.
```

stefsmeets commented on _episodes/conclusion.md:
```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
+## Improving FAIRness
+
+We provided recommendations, examples of steps you can take before, during and after your research project
+to make your research data more FAIR. FAIR sounds like a lot of work. However, you can get started by:
```
```suggestion
to make your research data more FAIR. FAIR sounds like a lot of work. However, here is how you can get started:
```

stefsmeets reviewed (changes requested):
Hi Sarah, nice work! I changed some of the verbs to make them more actionable.

Do you think it is possible to find some higher resolution images? The current ones are a bit grainy. And if possible, I think it would be better to put them in a different place, the current layout does not look great.

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
Update _episodes/conclusion.md

Co-authored-by: Stef Smeets <stefsmeets@users.noreply.github.com>

SarahAlidoost committed:
remove the figures

SarahAlidoost commented:
> Hi Sarah, nice work! I changed some of the verbs to make them more actionable.
> 
> Do you think it is possible to find some higher resolution images? The current ones are a bit grainy. And if possible, I think it would be better to put them in a different place, the current layout does not look great.

@stefsmeets thanks. Your suggestions are commit now. I agree with you on the figures. I replaced them with the one in the introduction. Is it better now?

stefsmeets commented:
> > Hi Sarah, nice work! I changed some of the verbs to make them more actionable.
> > Do you think it is possible to find some higher resolution images? The current ones are a bit grainy. And if possible, I think it would be better to put them in a different place, the current layout does not look great.
> 
> @stefsmeets thanks. Your suggestions are commit now. I agree with you on the figures. I replaced them with the one in the introduction. Is it better now?


Yep, awesome! I was trying to play with the html to make it fit, but the formatting is not so easy to fix in this template. 👍
